task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +528,33 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let gain; 
+
+    let verif = position.some((value, i, arr) => {     
+        gain = arr[i][0]; 
+        return (arr[i][0] === arr[i][1] && arr[i][1] === arr[i][2] && arr[i][0] !== undefined); 
+    }); 
+  
+    if (verif) {
+        return gain;
```

Comment:
I don't think **gain** means what you think it means. Try to come with a better name